src/components/themed/CryptoExchangeFlipInputWrapperComponent.js

```diff
@@ -33,8 +33,13 @@ export type Props = {
   children?: React.Node
 }
 
-class CryptoExchangeFlipInputWrapperComponent extends React.Component<Props & ThemeProps> {
+type State = {
+  error?: string
+}
+
+class CryptoExchangeFlipInputWrapperComponent extends React.Component<Props & ThemeProps, State> {
   launchSelector = () => {
+    this.onError()
```

Comment:
Why are you calling onError here and not only when you actually have an error?

Reply:
this.clearError()